pkg/reexec/reexec_test.go

```diff
@@ -51,7 +51,7 @@ func TestCommand(t *testing.T) {
 func TestCommandContext(t *testing.T) {
 	stdout := &bytes.Buffer{}
 
-	ctx, _ := context.WithDeadline(context.TODO(), time.Now().Add(5*time.Second))
+	ctx, finish := context.WithDeadline(context.TODO(), time.Now().Add(5*time.Second))
```

Comment:
Add `defer finish()` here, otherwise it's not called in case of an error and some linter is going to complain

Reply:
Fixed